### PR TITLE
Feature/scc 605 travis build

### PR DESCRIPTION
**Pull request by adrianmilne**

### CODE REVIEW

File: iac/environments/sbx1/backend.tf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+terraform {
+  backend "s3" {
+    region = "eu-west-2"
+    key    = "ccs-scale-agreements-service-sbx1"
```
Comment (tommyb82) [inline iac/environments/sbx1/backend.tf:4]: Just wondering - looks like we need to differentiate the keys in the backend files in `ccs-scale-shared-paas-infra` - and do we want to follow the same naming strategy (i.e. `namespace/[infra|service]` ?

File: iac/environments/sbx2/backend.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+terraform {
+  backend "s3" {
+    region = "eu-west-2"
+    key    = "ccs-scale-agreements-service-sbx1"
```
Comment (tommyb82) [inline iac/environments/sbx2/backend.tf:4]: As above - guess this needs to change

File: iac/modules/agreements-service/provider.tf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.14.2"
```
Comment (tommyb82) [inline iac/modules/agreements-service/provider.tf:5]: Could make this value a variable with a default in `deploy-all/variables.tf` and pass in to each module - save repeating it in each provider.tf?

**Review by tommyb82 — APPROVED** [review]
Awesome! Just those few little comments / questions.. 👍

File: iac/environments/sbx1/backend.tf (hunk 1 above)
Comment (tommyb82) [reply iac/environments/sbx1/backend.tf:4]: Ignore the above - I see you have updated all the keys in the shared infra PR - nice one 👍

File: iac/modules/agreements-service/provider.tf (hunk 3 above)
Comment (adrianmilne) [reply iac/modules/agreements-service/provider.tf:5]: I agree - that would be better - feels like there are improvements that could be made in these duplicated files. If ok - I will merge as-is for now in order to try and get the resources deployed asap, and improve as a follow on task

**Pull request merged**